src/components/Web3Status.tsx

```diff
@@ -41,9 +41,13 @@ const Web3Status = (): ReactElement => {
         endIcon={account && <Identicon />}
       >
         <Typography variant="body1" whiteSpace="nowrap">
-          {account
-            ? udName || ensName || shortenAddress(account)
-            : t("connectWallet")}
+          {isLoading ? (
+            <Typography>Loading...</Typography>
```

Comment:
Seems like don't need `typography` because it is already wrapped with `typography`.

Reply:
Ok, I forgot to remove this. We should get some sort of design for loading states.